src/main/java/dk/kb/datahandler/oai/OaiHarvestClient.java

```diff
@@ -231,11 +233,21 @@ protected PasswordAuthentication getPasswordAuthentication() {
                 }).build();
 
 
+        /* The Preservica 7 OAI-PMH endpoint is acting up. Per their documentation they say that they follow the standard
+            and makes Basic authorixation possible. However, the response does not contain a WWW-Authenticate header, when queried.
+
+            If using their acces endpoint to get an access-token then the service works. I got an access token by calling the following:
+            curl -X 'POST'   'DEVELSERVER'   -H 'accept: application/json'   -H 'Content-Type: application/x-www-form-urlencoded'   -d 'username=USERNAME&password=PASSWORD&cookie=false&includeUserDetails=false'
+            Changing DEVELSERVER, USERNAME and PASSWORD
 
+            I've written the integration test dk.kb.datahandler.oai.OaiHarvestClientIntegrationTest.testPreservicaSevenAuth
+            which makes it seem like this might be a problem in our end as the test parses.
+         */
         HttpRequest request = HttpRequest.newBuilder()                          
-                .uri(URI.create(uri))              
-                .header("User-Agent", "Java 11 HttpClient Bot")            
-                .header("Authorization", getBasicAuthenticationHeader(user, password))                
+                .uri(URI.create(uri))
+                //.header("Preservica-Access-Token", "b8844076-d96d-4fe1-b8fe-e0dd6bee5bbc")
```

Comment:
This does not look like a thing that should be in the code.

Reply:
Correct, I used the preservica access api to test, while I couldn't run the service. Somehow the access token could be used over VPN while basic auth couldn't